### DISCUSSION TITLE
Did anyone try  the latest hyprland version 0.42.0

### DISCUSSION THREAD

**Discussion by Acebasein** [end-4/dots-hyprland#734] [General]
Did anyone try the latest hyprland version 0.42.0 with these dot files? Any feedback, please

**Comment by NextStep-IM** [comment]
Working good. `fakefullscreeen` was removed and end-4 fixed it: #719

**Reply by Acebasein** [reply]
Glad to hear that. Thank you, @NextStep-IM
